core/src/main/resources/reference.conf

```diff
@@ -100,6 +100,30 @@ akka.kafka.consumer {
   # Interval for checking that transaction was completed before closing the consumer.
   # Used in the transactional flow for exactly-once-semantics processing.
   eos-draining-check-interval = 30ms
+
+  # Settings for connection checker. Connection checking using listTopics request with timeout
+  # configurated by akka.kafka.consumer.metadata-request-timeout
+  connection-checker {
+
+    #Flag to turn on connection checker
+    enable = false
+
+    # Fully qualified config path which holds the dispatcher configuration
+    # to be used by the KafkaConnectionChecker. Some blocking may occur.
+    dispatcher = ${akka.kafka.consumer.use-dispatcher}
+
+    # Ammount of attempts that would be perform after first connection failure occurs
+    # Required not negative number
+    max-retries = 3
+
+    # Intreval between listTopics response and new request. Also used as base for exponential retry
+    check-interval = 1s
```

Comment:
The default should be larger to align with the other defaults, maybe 15 seconds.